recipes/aws-c-sdkutils/all/conanfile.py

```diff
@@ -40,10 +40,12 @@ def layout(self):
         cmake_layout(self, src_folder="src")
 
     def requirements(self):
-        if Version(self.version) <= "0.1.3":
+        if Version(self.version) < "0.1.12":
             self.requires("aws-c-common/0.8.2", transitive_headers=True, transitive_libs=True)
+        elif Version(self.version) == "0.1.12":
+            self.requires("aws-c-common/[>=0.9.4 <=0.9.10]", transitive_headers=True, transitive_libs=True)
```

Comment:
```suggestion
            # [>=0.9.4 <=0.9.10]
            self.requires("aws-c-common/0.9.10", transitive_headers=True, transitive_libs=True)
```

Reply:
Thanks for pointing this out. I made the change accordingly.

Reply:
There is a minor difference. Because the current recipe of `was-c-common` doesn't have `0.9.10`, I use `0.9.6` here.